indico/modules/categories/templates/display/sidebar.html

```diff
@@ -0,0 +1,71 @@
+{% from 'attachments/_attachments.html' import render_attachments %}
+
+{% macro render_attachments(category) %}
+    {%- if category.attachment_count %}
+        {{ render_attachments(category.attached_items, linked_object=category,
+                              management=category.can_manage(session.user)) }}
+    {% endif -%}
+{% endmacro %}
+
+{% macro render_managers(managers) %}
+    {%- if managers %}
+        <h2 class="icon-medal">{% trans %}Managers{% endtrans %}</h2>
+        <ul id="manager-list">
+            {% for manager in managers %}
```

Comment:
prettier HTML, doesn't really matter, it gets ugly either way due to the use of macros, blocks, etc ;)